src/data/protocols/guardian/update-guardian-password-repository.ts

```diff
@@ -0,0 +1,8 @@
+export interface UpdateGuardianPasswordRepository {
+  updatePassword: (userData: UpdateGuardianPasswordRepository.Params) => Promise<UpdateGuardianPasswordRepository.Result>
+}
+
+export namespace UpdateGuardianPasswordRepository {
+  export interface Params { id: string, password: string }
+  export type Result = boolean
```

Comment:
Maybe boolean to Result type is not best approach. If has a false value we are asume that guardian not found, is it true every time? Maybe explicity this is more appropriate.